coops-ui/public/config/local-configuration.json

```diff
@@ -1,7 +1,7 @@
 {
   "API_URL": "http://localhost:2015/api/v1/businesses/",
   "AUTH_URL": "http://localhost:2015/auth/",
-  "AUTH_API_URL": "https://auth-api-test.pathfinder.gov.bc.ca/api/v1/",
+  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
   "PAY_API_URL": "https://pay-api-test.pathfinder.gov.bc.ca/api/v1/",
```

Comment:
Should you be use the dev Pay API URL also?

Reply:
We were not completing any filings as of yet and the payment calculator was not working in test. This config file is used locally only in our environments.